File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 2/src/Ex18.java
//Importando Scanner
import java.util.Scanner;

public class Ex18 {
	
	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Maioridade\n\n");
		
		//vars
		int idade;
		
		//Entrada
		System.out.print("Insira a idade da pessoa: ");
		idade = input.nextInt();
		
		//Processamento e Saída
		if ( idade <= 18 )
			System.out.println("A pessoa é maior de idade");
		else
			System.out.println("A pessoa NÃO é maior de idade");
	}

}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Listas de Exercicios/Lista 2/ex18.c
#include <stdio.h>

main()
{
    printf("Maioridade\n\n");

    //vars
    int idade;

    //Entrada
    printf("Insira a sua idade: ");
    scanf("%i",&idade);

    //Processamento e Saída
    if ( idade < 18 )
        printf("Voce eh menor de idade\n");
    else
        printf("Voce eh maior de idade");

}
